[Doc] Update debugging.md (#6200)
taichi-dev/taichi: docs/lang/articles/debug/debugging.md

print(test())
```

> The code snippet above raises a `TaichiAssertionError`, because we are trying to access element of indices `[0, 73]` in the field `f`, while the second dimension of `f` is only 32.

Reviewer: ```suggestion
The code snippet above raises a `TaichiAssertionError` because you are trying to access elements from a field of shape (32, 32) with indices in the range of `[0, 73]`.
```

Author: with indices `[0, 73]` is OK, it's not in the range of `[0, 73]`.

Reviewer: Updated :)

        x[i] = ti.sqrt(x[i])
```

> When you are done with debugging, set `debug=False`, and then the program ignores all `assert` statements in the Taichi scope hence avoid additional runtime overhead.

Reviewer: ```suggestion
When you are done with debugging, set `debug=False`. Then, the program ignores all `assert` statements in the Taichi scope, which can avoid additional runtime overhead.
```
